index.html

```diff
@@ -1,56 +1,188 @@
+				/>
+
+				<!-- Title -->
+				<div class="title">
+					<h1>Overview</h1>
+					<h4>Subtitle Text</h4>
+				</div>
+				<!-- Content -->
+				<h2>About This Site</h2>
+				<p>
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt
+					blanditiis earum voluptate obcaecati voluptates expedita, hic vero,
+					suscipit alias, quibusdam excepturi ullam exercitationem. In
+					reprehenderit dolorem aperiam dignissimos pariatur? Vitae?
+				</p>
+				<p>
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sapiente
+					nisi iure delectus quod optio cumque odit est modi, placeat ipsam
+					iusto fuga consequatur id commodi quos eos quaerat sunt consectetur.
+				</p>
+
+				<h2>About us</h2>
+				<p>
+					Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus
+					tempora ex voluptates harum distinctio suscipit voluptatum
+					necessitatibus ut veniam. Doloribus pariatur molestias accusamus
+					voluptate facilis impedit sequi tempore, culpa ex.
+				</p>
+
+				<img class="hhImage" src="images/image-placeholder.jpeg" alt="image" />
+
+				<h3>Haaga-Helia</h3>
+				<p>
+					Lorem ipsum, dolor sit amet consectetur adipisicing elit. Obcaecati
+					dicta vel, est eum alias cupiditate vero tempore eaque voluptates sunt
+					exercitationem facere praesentium, non impedit earum cumque architecto
+					optio neque? Learn more at
+					<a href="https://www.haaga-helia.fi/en">haaga-helia.fi</a>
+				</p>
+
+				<img
+					class="iupuiImage"
+					src="images/image-placeholder.jpeg"
+					alt="image"
+				/>
+				<h3>IUPUI</h3>
+				<p>
+					Lorem ipsum dolor, sit amet consectetur adipisicing elit. Corrupti,
+					dicta sint. Itaque pariatur amet impedit voluptatem eveniet accusamus,
+					voluptatum placeat aperiam nulla quos tempora culpa aliquid
+					consequatur. Quibusdam, iste doloribus?
+				</p>
+
+				<h2>Helpful Resources</h2>
+				<p>
+					Lorem ipsum dolor sit amet consectetur adipisicing elit. Nihil
+					mollitia, itaque quis laborum quos veniam libero repellendus deleniti
+					ex similique harum praesentium aut voluptatibus suscipit voluptates?
+					Maiores inventore obcaecati sed?
+				</p>
+
+				<ul>
+					<li>
+						<a
+							href="https://www.google.com/webhp?hl=en&sa=X&ved=0ahUKEwjcoP-DppT3AhX8QvEDHbekAwwQPAgI"
+							class="resource"
+							>google.com</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://www.google.com/webhp?hl=en&sa=X&ved=0ahUKEwjcoP-DppT3AhX8QvEDHbekAwwQPAgI"
+							class="resource"
+							>google.com</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://www.google.com/webhp?hl=en&sa=X&ved=0ahUKEwjcoP-DppT3AhX8QvEDHbekAwwQPAgI"
+							class="resource"
+							>google.com</a
+						>
+					</li>
+				</ul>
+			</main>
+
+			<!-- Footer -->
+			<footer class="footer">
+				<a href="#" class="btt">Back to Top</a>
+				<a href="index.html"
+					><img src="images/logo-placeholder.png" alt="logo"
+				/></a>
+				<p>Copyright 2022</p>

```

Comment:
If we want to be fancy we could dynamically insert the year via JavaScript  

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getYear